build/scripts/create-tags

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# Important: This script should never cause the pipeline to fail, so that the tags can be created outside of it.
+# If run locally, it:
+# - does not need to run inside pmm-submodules as it clones all the necessary repos
+# - does not need to configure git to push using ssh, this user should set it all up ahead of time
+# - will use the user's creds and email to tag the repos, therefore sufficient permissions are required
+
+set +o errexit
+set +o nounset
+set -o xtrace
+
+# List of repositories whose release branches need to be tagged
+declare repos=(
+    "percona-platform/grafana"
+    "percona/grafana-dashboards"
+    "percona/pmm"
+    "Percona-Lab/pmm-submodules"
+)
+
+# These setting are only needed when running in CI (Jenkins or github actions)
+if [ -n "$CI" ]; then
+  # Configure git settings globally
+  git config --global advice.detachedHead false
+  git config --global user.email "noreply@percona.com"
+  git config --global user.name "PMM Jenkins"
+
+  # Configure git to push using ssh
+  export GIT_SSH_COMMAND="/usr/bin/ssh -i ${SSHKEY} -o StrictHostKeyChecking=no -o LogLevel=error -o UserKnownHostsFile=/dev/null"
+fi
+
+TAG="v${VERSION}"
+echo "We will be tagging repos with a tag: $TAG"
+
+for REPO in "${repos[@]}"; do
+    DIR_NAME=${REPO##*/} # grab the repo name from <company>/<repo>
+    rm -fr "$DIR_NAME" || true
+    mkdir -p "$DIR_NAME"
+
+    pushd "$DIR_NAME" >/dev/null
+        git clone https://github.com/${REPO} ./
+        # The default is https, and we want to set it to ssh
+        git remote set-url origin git@github.com:${REPO}.git
+
+        BRANCH="pmm-${VERSION}"
```

Comment:
we are planning to change our branching logic from `2.39.0` to `2.39.x` where `x` is just symbol `x`, not a number, so this code will be outdated pretty soon. In Python script we were relying on pmm-submodules which uses git submodule and knows exact commit which was used

Reply:
I see no value added if all that Go has to do is to run the very same commands using "os/exec". We can certainly use submodules for this. Should we?

Reply:
Yes, I think. if we will migrate to new branching logic it will make more sense

Reply:
Are you saying we will tag releases with `2.40.x`? How is that?

Reply:
no, branch name will be 2.40.x, but tags will be 2.40.0, 2.40.1, 2.40.2, etc...

Reply:
Done.